webview/src/plots/components/templatePlots/TemplatePlotsWrapper.tsx

```diff
@@ -20,6 +21,7 @@ export const TemplatePlotsWrapper: React.FC = () => {
       changeSize={changeSize}
       hasItems={hasItems}
       height={height}
+      addPlotsButton={{ onClick: addPipelinePlot }}
```

Comment:
Tried to add a plot button to the grid, but couldn't get past the drag and drop bugs (I needed the button to stay at the end of the plots but trying to drag and drop plots around the button would break things). There is most likely a way to implement it, but I was definitely having a hard time understanding the plots drag and drop code enough to figure out how. 

I'll give it another try as a followup later.

Reply:
Quick feedback on the UX (which I assume I missed in a previous PR).

Clicking on this button presents the user with this screen:

![image](https://github.com/iterative/vscode-dvc/assets/37993418/90ba510d-723c-4660-aae5-8fc3a6b1cfc7)

It is not obvious from this screen (unless you have context like you and I do) that the user is looking to add a file that contains data that can be used to generate a plot.

Reply:
Also in the demo the tooltip says "Add Plot**s**" but should be "Add Plot":

![image](https://github.com/iterative/vscode-dvc/assets/37993418/9e256dde-5717-43cd-b206-92f438c93cc8)

Reply:
> It is not obvious from this screen (unless you have context like you and I do) that the user is looking to add a file that contains data that can be used to generate a plot.

Oh, good point! First idea that comes to mind is opening a quick pick that lets user type in a relative path or select a file (we do something similar when creating a train stage). The quick pick can add a sentence that would clarify things further. I'll update #4654.